src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
```

Comment:
So removing things that we're not using is a good idea. I couldn't find a page that used the subscriptions block either. I'm unclear on what we're doing for the office pages that are published, but aren't "officially" published. For instance, http://beta.consumerfinance.gov/offices/office-of-financial-education/ uses the media images, but isn't an officially published page, yet still is live. It would have to be tested, but removing the resource icon block could break the layout on that page. cc @schaferjh